Update off and unavailable in standby state (#2728)
home-assistant/developers.home-assistant: docs/core/integration-quality-scale/rules/entity-unavailable.md

## Exceptions

> If an integration has a way to turn on a device, either via a user-defined automation trigger or by automatically creating a secondary control channel (e.g., using Wake On Lan or Infra-Red blaster) - then the device should be reported as `off` when it is in standby and unresponsive to the main channel (e.g., TCP).
> If no such method exists, and the device cannot be controlled in its current state, it should be reported as `unavailable`.

Reviewer: ```suggestion
If an integration can turn on a device, either via a user-defined automation trigger or by automatically creating a secondary control channel (e.g., using Wake-on-LAN or infrared blaster), then the device should be reported as `off` when it is in standby and unresponsive to the main channel (e.g., TCP). If no such method exists, and the device cannot be controlled in its current state, it should be reported as `unavailable`.
```
